### ISSUE TITLE
Performing multitouch swipe gesture until last one finger is released

### ISSUE THREAD

**Issue by Polisson** [JoseExposito/touchegg#454]
Hi,
What I describe below is a bug for me, but I understand that would be kind of an ability for some users:

When multitouch gesture is started (with action at gesture end), releasing a finger (3rd or 4th) ends process and triggers assigned action. The four-finger gesture activates when the fourth finger looses contact with the surface and three.

Much more convenient and intuitive would be ending a gesture process on releasing the very last finger. 
In other words, we could start the gesture with 3 (or 4) fingers, but till the last one finger is in touch with the surface, the gesture is ongoing.

BTW that is default behavior on macOS, but in your tool would be an option.

*P.S. Big thanks Jose, for Your really great work! Really awesome - I just can't wait for Elementary OS stable release, where your piece of software shines! 8D*

**Comment by JoseExposito** [comment]
Hi! And thanks for your bug report and your kind words.

Touchégg relies on libinput to identify gestures, and that's how a gesture is defined there:
https://wayland.freedesktop.org/libinput/doc/latest/gestures.html#lifetime-of-a-gesture

Unfortunately, I can not change this behaviour.

I agree with you, I think that's a better experience. Actually, on mulitouch screens, where libinput doesn't provide gestures and I have to interpret them from coordinates, that's how it works.

Closing the issue as it is not possible to fix it

**Comment by Polisson** [comment]
Okay, I understand your point...
...but let me clarify it: you are saying that Touchegg, after libpinput gesture ends, can't "pretend" the action till the last finger is lifted?
Is it so 1:1 relationship?  
Don't get me wrong, I don't question Your idea or implementation. Only need to know if it's a matter enhancing libinput (making feature requests or contributing to code) or a matter of enhancing **Your idea**. Furthermore, really useful implementation of gestures need one more thing which is a momentum of performed gesture. If so, would it be done with Touchegg development?
Please, explain it to me, really appreciate.
Greetings and thanks in advance.

**Comment by JoseExposito** [comment]
> you are saying that Touchegg, after libpinput gesture ends, can't "pretend" the action till the last finger is lifted?

Exactly, it is absolutely impossible to continue the action with the current libinput API. Basically because the API is not reporting the required information.

I don't think it is something the libinput devs will want to change though, because the current behaviour is well documented by them here:
https://wayland.freedesktop.org/libinput/doc/latest/gestures.html#lifetime-of-a-gesture
